qlib/kernel/threadmgr/task_exit.rs

```diff
@@ -912,7 +912,10 @@ impl Thread {
 
         let creds = self.Credentials();
         let kuid = creds.lock().RealKUID;
+        #[cfg(not(feature = "cc"))]
         let userns = receiver.UserNamespace();
+        #[cfg(feature = "cc")]
+        let userns = creds.lock().UserNamespace.clone();
```

Comment:
I do not call this function, I use the attribute directly.

Reply:
> I do not call this function

hmmm I missed the `()` .. but anyways why not just call it?

Reply:
> > I do not call this function
> 
> hmmm I missed the `()` .. but anyways why not just call it?
As the commit log, to avoid deadlock. Actually I don't know why the deadlock happened, but the lock() inside UserNamespace() causes a deadlock.